pkg/util/jsontesting/json_roundtrip.go

```diff
@@ -22,9 +22,28 @@ import (
 	"reflect"
 	"strings"
 
+	jsonv2 "k8s.io/kube-openapi/pkg/internal/third_party/go-json-experiment/json"
+
+	"github.com/go-openapi/jsonreference"
 	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 )
 
+func JsonCompare(got, want []byte) error {
+	if d := cmp.Diff(got, want, cmp.Transformer("JSONBytes", func(in []byte) (out interface{}) {
+		// Note that integers larger than 2^53 will lose precision
+		// since they will be represented using a Go float64.
+		// There are workarounds for this, but calling it out.
+		if err := json.Unmarshal(in, &out); err != nil {
```

Comment:
`sigs.k8s.io/json` has `UnmarshalCaseSensitivePreserveInts` which will preserve ints... do we want to use that?

Reply:
I don't have a strong opinion, this is strictly with-in the test and we're not having any problems with it?

Reply:
done